src/test/resources/test-input.xml

```diff
@@ -14,6 +14,19 @@ http://www.mulesoft.org/schema/mule/api-platform-gw http://www.mulesoft.org/sche
 
     <!--Here's a comment surrounded wrapped by empty lines to make sure they are kept-->
 
+    <!--
+    Here's a comment block.
+    Make sure they have correct indentation after format.
+    -->
+
+    <!--
+        Here's a comment block with leading spaces.
+        Make sure they have correct indentation and keep the leading spaces after format.
+    -->
+
+    <!--Here's a comment block with unaligned block start/end.
+    Make sure they have correct indentation after format.-->
+
```

Comment:
These input lines look the same as the output lines, so the formatter doesn't actually change anything. It would be good to improve these test cases and indent these a bit erratically to ensure that the formatter indents them as expected in the output.

I tried to indent some of these, and it seemed to work okay, but the "unaligned block" case was very unpredictable. While changing the indent of the first line in that case seemed to result in formatted output that returned it to something sane, I found that I couldn't make any changes to the indent of the second line that resulted in formatter output that made sense.

Reply:
thanks for your comments, I'll do a fix later.